site-specific/mays/transfer_check_mays.py

```diff
@@ -29,7 +32,7 @@
 
 from pymedphys._experimental.chartchecks.dvh_helpers import plot_dvh
 
-
+streamlit_analytics.start_tracking()
```

Comment:
Keen to hear how it goes :)

Reply:
I'm having some issues when trying to implement the analytics into the app run in the pymedphys gui. I'm sure there's a simple reason that I'm missing. Also I'm not sure if this will help me exactly with what I need. I was hoping to have a way to time results, and this just seems to show traffic without any time information (again, unless I'm missing something).

Reply:
Given Streamlit's written in Python, essentially "anything" is possible. So there certainly is a way to manipulate things into getting the timing information you want. The only question is how complicated it needs to be to make it happen.

Would you be able to detail what it is exactly you're after and I'll see if I can think of a neat solution.

Reply:
I guess what would be ideal is just a time starts as soon as the file is uploaded for the transfer check, and then the user can click a button at the bottom once they've finished to write the amount of time it took them to complete the transfer check. I believe my problem when I tried this a while ago was that the start time would change any time the user did anything to update the page.